litmus-portal/frontend/cypress/component/customSlider.spec.tsx

```diff
@@ -0,0 +1,84 @@
+/// <reference types="Cypress" />
+import React, { useState } from 'react';
+import { mount } from 'cypress-react-unit-test';
+
+import CustomSlider from '../../src/components/CustomSlider/index';
+describe('Custom Slider with value 10', () => {
+  it('is visible', () => {
+    const wrapper = (
+      <CustomSlider testName="Test" value={10} onChangeCommitted={() => {}} />
+    );
+    mount(wrapper);
+    expect(wrapper.props.value).to.equal(10);
+  });
+
+  it('The slider is enabled', () => {
+    cy.get('.MuiSlider-root').first().log('Slider Enabled');
```

Comment:
Can you explain what are you checking here?If you are checking something, then log any details.

Reply:
here, I'm checking if the slider is mounting properly

Reply:
Okk then just a suggestion, first check like 
cy.get().first().should(). //Any assertion
cy.log()

Reply:
Thanks for the sugesstion @Jonsy13 . I'll keep that in mind!